homework-submission/week1-react/src/App.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
+import todoLogo from './todo.png';
+import './App.css';
+import Todo from "./todo.js";
+import data from './todoData.json';
+
+class App extends Component {
+
+  state = {
+    title: 'My Todo List',
+    data: data
+  };
+
+  changeDone = (index) => {
+    const newState = this.state;
+    if(this.state.data[index].done) {
+      newState.data[index].done = false  
+    } else {
+      newState.data[index].done = true
```

Comment:
Actually I dont need to use `.map` because I change just one item's `done` ' s value. If it is false, when I click the checkbox, it should be true, and vice versa.